enferno/admin/views.py

```
@@ -3126,6 +3122,13 @@
     )
 
 
+# Bulletin fields routes
+@admin.route("/bulletin-fields/", defaults={"id": None})
+def bulletin_fields(id: Optional[t.id]) -> str:
```

Ruff (0.12.2)

Unused function argument: `id`

(ARG001)